app/src/main/java/app/foodfinderapp/ui/viewModel/UserViewModel.kt

```diff
@@ -48,23 +48,17 @@ class UserViewModel: ViewModel() {
         locationLiveData.value = token
     }
 
-    //密码登录
+
     val passwordLogin = Transformations.switchMap(passwordLoginLiveData){ passwordLogin ->
         Repository.getPasswordLogin(passwordLogin.Phone, passwordLogin.password)
     }
 
-    fun resultPasswordLocation(phone: String, password: String, Email: String){
-        passwordLoginLiveData.value = PasswordLoginArgs(phone, password, Email)
+    fun resultPasswordLocation(phone: String, password: String, email: String){
+        passwordLoginLiveData.value = PasswordLoginArgs(phone, password, email)
     }
 
-    //修改信息
-//    fun reviseAddress = Transformations.switchMap(reviseData){  reviseData->
-//        Repository.reviseAddress(reviseData.message,reviseData.code)
-//    }
-//
-//    fun resultReviseAddress(){
-//        reviseData.value =
-//    }
```

Comment:
Good suggestion.


Avoid pushing commented code to Version Control.  Version control gives us better options:
- You can look at history to see what used to be there.
- You can use branches for experimental features.